app/assets/stylesheets/earthworks.css

```diff
@@ -141,4 +141,38 @@ Uncomment lines 142-144
 
 /* .search-query-form .search-btn .blacklight-icons-search {
   display: none;
-} */
\ No newline at end of file
+} */
+
+.filter .btn-outline-secondary {
+  --bs-btn-border-color: var(--stanford-fog-light);
+  --bs-btn-color: var(--stanford-black);
+  background-color: var(--stanford-fog-light);
```

Comment:
Why do you need this?  Can you set `--bs-btn-bg` instead?

Reply:
I could swear that didn't work when I tried it, but it does. Now changed.